### PR TITLE
Update continent thickness, orogeny intensity + misc fixes

### PR DESCRIPTION
**Pull request by pjanik**
https://www.pivotaltracker.com/story/show/181254099

The main goal of this PR is to update continental crust and mantle thickness. 

Old:
Screenshot 2022-02-24 at 14 47 26
New:
Screenshot 2022-02-24 at 14 47 30

The mantle is easy, as it's just a visual layer. However, there's no single way to update continental thickness without affecting the model output. I've done two things:

1. I've changed the ratio of "mountain roots" (CRUST_THICKNESS_TO_ELEVATION_RATIO). This makes continental crust thinner, as its base elevation is unchanged (0.55). There's less of the crust below zero level.
2. I've shifted base ocean crust elevation a bit higher. That makes oceans shallower and decreases the total height of the continental crust.

However, both these updates make mountains taller, as more of the crust will "float" and also the base elevation is higher. That's why I've been testing and updating other params that affect orogeny strength.

I've also updated presets to use new force values and fixed force arrows in GEODE mode.

### CODE REVIEW

File: js/config.ts [hunk 1]
```diff
@@ -85,7 +85,7 @@ const DEFAULT_CONFIG = {
   oceanDensity: 1,
   continentDensity: 3,
   // How fast continent is stretching along divergent boundary. Bigger value means it would turn into ocean / see faster.
-  continentalStretchingRatio: 8,
+  continentalStretchingRatio: 4,
```
Comment (kswenson) [inline js/config.ts:88]: Out of curiosity, should the second-to-last word in the comment in line 87 be "sea"?

File: js/plates-model/crust.ts [hunk 2]
```diff
@@ -45,16 +45,20 @@ export const MIN_LAYER_THICKNESS = 0.02 * BASE_OCEANIC_CRUST_THICKNESS;
...
+export const MAX_WEDGE_SEDIMENT_THICKNESS = 5 * MAX_REGULAR_SEDIMENT_THICKNESS;
 export const WEDGE_ACCUMULATION_INTENSITY = 6;
 
 // When the crust subducts, most of its rock layers are transferred to neighboring fields.
 // When this value is low, it will be transferred slower than subduction and most of the rock will be lost.
 // When this value is high, pretty much all the rocks will be redistributed to non-subducting neighbors.
-export const ROCK_SCARPING_INTENSITY = 50;
+export const ROCK_SCARPING_INTENSITY = 40;
 
+// These two constants decide about orogeny intensity.
```
Comment (kswenson) [inline js/plates-model/crust.ts:56]:
```suggestion
// These two constants determine orogeny intensity.
```

File: js/plates-model/crust.ts [hunk 3]
```diff
@@ -45,16 +45,20 @@ export const MIN_LAYER_THICKNESS = 0.02 * BASE_OCEANIC_CRUST_THICKNESS;
...
+// ROCK_FOLDING_INTENSITY adds some variability to mountains.
 export const ROCK_FOLDING_INTENSITY = 15;
-export const ROCK_TRANSFER_INTENSITY = 25;
+// ROCK_TRANSFER_INTENSITY moves rocks form subducting plate to the overriding and speed of this transfer is the
```
Comment (kswenson) [inline js/plates-model/crust.ts:59]:
```suggestion
// ROCK_TRANSFER_INTENSITY moves rocks from subducting plate to the overriding and speed of this transfer is the
```

**Review by kswenson — APPROVED** [review]
👍 LGTM -- I'll trust you and the project team to assess whether the parameter changes have the desired effect. The code changes make sense and I appreciate the additional comments. I just made a few minor suggestions on some of the comments.

File: js/config.ts (hunk 1 above)
Comment (pjanik) [reply js/config.ts:88]: It should. :) I'll fix it.

**Pull request merged**